protocol/src/lib.rs

```diff
@@ -0,0 +1,126 @@
+/// The default protocol used by the signalling server
+use serde::{Deserialize, Serialize};
+
+#[derive(Serialize, Deserialize, Debug, PartialEq)]
+#[serde(tag = "peer-type")]
+#[serde(rename_all = "lowercase")]
```

Comment:
`peer-type` with dash but `lowercase` would cause other things to have an underscore

Reply:
You probably want `snake_case` or `kebab-case` instead of `lowercase` and then make it all consistent everywhere.

Reply:
ended up going camelCase for consistency `event.candidate.toJSON()` on the javascript side